Add FAIR Guide with Checklists (#37)
Imageomics/Imageomics-guide: docs/wiki-guide/FAIR-Guide.md

# FAIR Guide

> This section provides information and resources to help ensure that digital products are ***F***indable ***A***ccessible ***I***nteroperable ***R***eusable and Reproducible[^1]. A general [Metadata Checklist](Metadata-Checklist.md) is provided to start one thinking about the type of information to be collected. Additionally, we include checklists for [code](Code-Checklist.md), [data](Data-Checklist.md), and [model](Model-Checklist.md) repositories. The code checklist focuses on the contents of a well-documented GitHub repository, while the data and model checklists cover the content of the [data](HF_DatasetCard_Template_mkdocs.md/) and [model](HF_ModelCard_Template_mkdocs.md/) card templates, respectively.

Reviewer: ```suggestion
This section provides information and resources to help ensure that digital products are ***F***indable, ***A***ccessible, ***I***nteroperable, ***R***eusable, and Reproducible[^1]. A general [Metadata Checklist](Metadata-Checklist.md) is provided to stimulate thinking about the type of information to be collected. Additionally, we include checklists for [code](Code-Checklist.md), [data](Data-Checklist.md), and [model](Model-Checklist.md) repositories. The code checklist focuses on the contents of a well-documented GitHub repository, while the data and model checklists cover the content of the [data](HF_DatasetCard_Template_mkdocs.md/) and [model](HF_ModelCard_Template_mkdocs.md/) card templates, respectively.
```

The last topic in this section discusses different methods of [DOI Generation](DOI-Generation.md) for digital products (code, data, and models). It focuses on our selected method for dataset publication: [Hugging Face](https://huggingface.co/), with some guidance on using [Zenodo](https://zenodo.org/) to archive code (specifically, a GitHub repository). For more information about other common data publication venues&mdash;and to see the thought process behind our selection&mdash;see the [Data Archive Options Comparative Overview](../pdfs/Data_Archive-Publication-Options-Comparative-Overview.pdf) for more information.[^3] Generating a DOI for a digital product is part of ensuring a globally unique and persistent identifier that can be used to reference and refer back to a digital product&mdash;an important component of FAIR and Reproducible principles.

!!! info "References and Background"

>     If you want to learn more about FAIR and Reproducible principles, explore these resources that we used when creating this guide:

Reviewer: ```suggestion
    If you want to learn more about FAIR and Reproducible principles, explore these resources that we used when developing this guide:
```

    - Wilkinson, M., Dumontier, M., Aalbersberg, I. _et al._ The FAIR Guiding Principles for scientific data management and stewardship. _Sci Data_ **3**, 160018 (2016). [10.1038/sdata.2016.18](https://doi.org/10.1038/sdata.2016.18)
    - Barker, M., Chue Hong, N.P., Katz, D.S. _et al._ Introducing the FAIR Principles for research software. _Sci Data_ **9**, 622 (2022). [10.1038/s41597-022-01710-x](https://doi.org/10.1038/s41597-022-01710-x)
    - Balk, M. A., Bradley, J., Maruf, M., Altintaş, B., Bakiş, Y., Bart, H. L. Jr, Breen, D., Florian, C. R., Greenberg, J., Karpatne, A., Karnani, K., Mabee, P., Pepper, J., Jebbia, D., Tabarin, T., Wang, X., & Lapp, H. (2024). A FAIR and modular image-based workflow for knowledge discovery in the emerging field of imageomics. _Methods in Ecology and Evolution_, 15, 1129–1145. [10.1111/2041-210X.14327](https://doi.org/10.1111/2041-210X.14327)
    - The [FARR Research Coordination Network](https://www.farr-rcn.org/) has a number of interesting resources and events.

>     - The [Research Data Aliance for Interdisciplinary Research](https://www.rd-alliance.org/disciplines/rda-for-interdisciplinary-research/) also provides links to resources and events particularly focused on considerations in interdisciplinary research.

Author: When I remove the bullet, it simply aligns with the first paragraph of the top point (as in the Turing Way note). Adding a `>` gives the Ozoani subpoint. 
<img width="708" alt="Screenshot 2025-04-11 at 5 42 21 PM" src="https://github.com/user-attachments/assets/c32c9254-7b1b-4152-8d76-861fe2517b24" />
While a double indent puts it into a code block:
<img width="725" alt="Screenshot 2025-04-11 at 5 46 14 PM" src="https://github.com/user-attachments/assets/3b1ce719-c567-48d8-953e-800a294ef154" />

>     - The [Research Data Aliance for Interdisciplinary Research](https://www.rd-alliance.org/disciplines/rda-for-interdisciplinary-research/) also provides links to resources and events particularly focused on considerations in interdisciplinary research.

Author: Another option would be some kind of emoji in place of the point (these are `:sparkles:` and `:dizzy:`):
![Screenshot 2025-04-22 at 4 06 55 PM](https://github.com/user-attachments/assets/e6da5423-455c-4308-b529-8f8ad04346f5)

>     - The [Research Data Aliance for Interdisciplinary Research](https://www.rd-alliance.org/disciplines/rda-for-interdisciplinary-research/) also provides links to resources and events particularly focused on considerations in interdisciplinary research.

Reviewer: I think I'd simply use italics to signify that it's commentary, rather than a quote (which the `>` would suggest) or something to do with AI (such as :sparkles: :wink:) or space travel (such as 💫 🫣 )

>     - The [Research Data Aliance for Interdisciplinary Research](https://www.rd-alliance.org/disciplines/rda-for-interdisciplinary-research/) also provides links to resources and events particularly focused on considerations in interdisciplinary research.

Author: Trying out the italics seems a bit odd. I think at this point I'd be inclined to just remove the bullet point as in the [first example](https://github.com/Imageomics/Imageomics-guide/pull/37#discussion_r2040334651) or try to find an "info" icon to replace the point (it seems overkill to do a nested info admonition).
![Screenshot 2025-04-23 at 2 34 25 PM](https://github.com/user-attachments/assets/99a9e436-4f40-4e8f-a547-63e2fa35763b)

>     - The [Research Data Aliance for Interdisciplinary Research](https://www.rd-alliance.org/disciplines/rda-for-interdisciplinary-research/) also provides links to resources and events particularly focused on considerations in interdisciplinary research.

Author: Definitely! Cycled through all the variations with @NetZissou and I'm on board with italics--it's about the best way to convey the information to a broad audience and I've been looking at this too long, so having the team weigh in is very helpful :)

!!! question "[Questions, Comments, or Concerns?](https://github.com/Imageomics/Imageomics-guide/issues)"

[^1]: While "Reproducible" is not part of the original FAIR principles as defined by the [Go-FAIR Initiative](https://www.go-fair.org/fair-principles/), we include it here to emphasize the importance of computational reproducibility alongside data stewardship. This extension reflects emerging practice in data-intensive science, where code, models, and workflows must be reusable and verifiable to support robust scientific claims. It is not part of the formal FAIR acronym, but aligns with broader community goals for open and transparent research.
[^2]: Full reproducibility is difficult to achieve; this [presentation](https://drive.google.com/file/d/1BFqZ00zMuyVHaD9A8PvzRDEg7aV0kp3W/view?usp=drive_link) by Odd Erik Gundersen provides a discussion of the varying degrees of reproducibilityand useful references when considering the level of reproducibility achieved by a given project.  

> [^3]: The [Data Archive Options Comparative Overview](../pdfs/Data_Archive-Publication-Options-Comparative-Overview.pdf) was created in May 2023 when we were deciding Institute archive recommendations, so it does not include information about newer features such as [Hugging Face's dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer), which greatly simplifies previewing datasets for downstream users.

Reviewer: ```suggestion
[^3]: The [Data Archive Options Comparative Overview](../pdfs/Data_Archive-Publication-Options-Comparative-Overview.pdf) was created in May 2023 as part of developing archive recommendations for the Institute, so it does not include information about newer features such as [Hugging Face's dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer), which greatly simplifies previewing datasets for downstream users.
```
